src/cloud-api-adaptor/Dockerfile.openshift

```diff
@@ -0,0 +1,22 @@
+# Use OpenShift golang builder image
+FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9-1737480393 as builder
+
+USER root
+RUN dnf install -y libvirt-devel
+
+COPY . /workdir 
+WORKDIR /workdir
+
+RUN export GOFLAGS="" && \
+    CGO_ENABLED=1 GOOS=linux go build \
+    -tags=strictfipsruntime,aws,azure,ibmcloud,vsphere,libvirt \
```

Comment:
do you mind adding gcp here ?

Reply:
The modified version has gcp in the list